packages/mongodb-artifact-generator/context/release-notes/atlas-cli-releaseInfo.yaml

```diff
@@ -0,0 +1,35 @@
+github:
+  owner: "mongodb"
+  repo: "mongodb-atlas-cli"
+  version: "atlascli/v1.22.0"
+  previousVersion: "atlascli/v1.21.0"
```

Comment:
what does this correspond to? a tag?

Reply:
Yeah they're git tags. Atlas CLI uses tags for releases so in the scope of this POC we're broadly assuming everyone does that